pkg/factory/risingwave_object_factory_test.go

```diff
@@ -186,19 +186,19 @@ func componentLabels(risingwave *risingwavev1alpha1.RisingWave, component string
 	}
 	if group != nil {
 		labels[consts.LabelRisingWaveGroup] = *group
-	}
-
-	if component == consts.ComponentFrontend {
+		labels = mergeMap(labels, risingwave.Spec.Global.Metadata.Labels)
+	} else if component == consts.ComponentFrontend {
 		labels = mergeMap(labels, risingwave.Spec.Global.ServiceMeta.Labels)
 	}
```

Comment:
The service meta shouldn't be included in the frontend Pods' metadata. I'm afraid there's a bug.

Reply:
Can you start another PR to fix that?